Description: Use x-www-browser instead of firefox
Author: Joachim Breitner <nomeata@debian.org>
Origin: vendor
Bug-Debian: http://bugs.debian.org/553330
Forwarded: not-needed
Last-Update: 2012-07-07
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
Index: b/XMonad/Prompt/Shell.hs
===================================================================
--- a/XMonad/Prompt/Shell.hs
+++ b/XMonad/Prompt/Shell.hs
@@ -228,7 +228,7 @@ env variable fallthrough = getEnv variab
    or pass it to a shell; so in that case, define $BROWSER as something like \"xterm -e elinks\"
    or as the name of a shell script doing much the same thing. -}
 getBrowser :: IO String
-getBrowser = env "BROWSER" "firefox"
+getBrowser = env "BROWSER" "x-www-browser"
 
 -- | Like 'getBrowser', but should be of a text editor. This gets the $EDITOR variable, defaulting to \"emacs\".
 getEditor :: IO String
